docs/data/rpc/api-reference/methods/getLedgerEntries.mdx

```diff
@@ -3,28 +3,125 @@ hide_title: true
 description: Returns ledger entries
 ---
 
+import { CodeExample } from "@site/src/components/CodeExample";
 import { RpcMethod } from "@site/src/components/RpcMethod";
 import rpcSpec from "@site/static/stellar-rpc.openrpc.json";
 
 <RpcMethod
   method={rpcSpec.methods.filter((meth) => meth.name === "getLedgerEntries")[0]}
 />
 
-### Generating `keys` Parameters
+The Stellar ledger is, on some level, essentially a key-value store. The keys are `LedgerKey`s and the values are `LedgerEntryData`s in the XDR. An interesting product of the store's internal design is that the key is a _subset_ of the entry: we'll see more of this later.
 
-The example above is querying a deployment of the [`increment` example contract] to find out what value is stored in the `COUNTER` ledger entry. This value can be derived using the following code snippets. You should be able to extrapolate from the provided examples how to get `keys` parameters for other types and values.
+The `getLedgerEntries` method returns the "values" (or "entries") for a given set of "keys". `LedgerKey`s come in a lot of forms, and we'll go over the valuable ones on this page alongside tutorials on how to build and use them.
 
-#### Python
+## Types of `LedgerKey`s
 
-:::note
+The source of truth should always be the XDR defined in the protocol. `LedgerKey`s are a union type defined in [Stellar-ledger-entries.x](https://github.com/stellar/stellar-xdr/blob/529d5176f24c73eeccfa5eba481d4e89c19b1181/Stellar-ledger-entries.x#L600). There are 10 different forms a ledger key can take:
 
-If you are using the [Python](https://stellar-sdk.readthedocs.io/en/latest/) `stellar_sdk` to generate these keys, you will need to install the latest version of the SDK. This can be done like so:
+1. **Account:** holistically defines a Stellar account, including its balance, signers, etc. (see [Accounts](https://developers.stellar.org/docs/learn/fundamentals/stellar-data-structures/accounts))
```

Comment:
At the beginning of the list, the list is describe as "forms a ledger key can take," however the list describes the ledger entries, rather than the keys. For example:

> **Account:** holistically defines a Stellar account, including its balance, signers, etc.

Maybe:

> There are 10 different forms a ledger key can take and each refers one-to-one with the following ledger entries: